docs/antctl.md

```diff
@@ -803,25 +803,29 @@ For more information about route advertisement, please refer to [Advertisements]
 # Get the list of all advertised bgp routes
 $ antctl get bgproutes
 
-ROUTE
-10.96.10.10/32
-192.168.77.100/32
-fec0::10:96:10:10/128
-fec0::192:168:77:100/128
+ROUTE                    TYPE                  K8s-OBJ-REF
+10.244.1.0/24            NodeIPAMPodCIDR       Pod
```

Comment:
let's remove `Pod` for the Pod CIDR ones
there is just no K8s object in that case, so let's keep it empty